deploy-with-init-params.md

```diff
@@ -20,51 +75,72 @@ Do not forget to stop it once you are done developing
 forgae node --stop
 ```
 
-## Step 1. Update your example project
-Lets add some state and init parameters to our example contract.
+## Step 1: Update your ExampleContract.aes
+
+Lets add some state and init parameters to our example contract which can be found at **contracts/ExampleContract.aes**
+
 ```
 contract ExampleContract =
-  record state = 
-    { savedNumber : int }
 
-  public stateful function init(num : int) = 
-    { savedNumber = num }
+  record state = { savedString : string }
```

Comment:
why change from int to string?

Reply:
I changed from int to string because I believe once someone understand string he/she can always use int

Reply:
Do you think I should change it back to int?

Reply:
Okay, should be fine, was just wondering why it was changed

Reply:
would you rename variables and function names from  pascalCase to snake case: `savedString` to `saved_string`